.devcontainer/Dockerfile.dev

```diff
@@ -0,0 +1,19 @@
+FROM python:3.9-slim
```

Comment:
As I understand it, we will not need a production app container for this project, so I intentionally omitted `Dockerfile`. My thinking is that this project will produce a published Python package for `benefits` and `eligibility-server` to consume.

If any of that is incorrect, let me know!

Reply:
That's definitely correct! Docker is only used for development in this case. I might actually rename the Dockerfile to `Dockerfile` since that is more canonical when you only have 1.